index.html

```diff
@@ -197,6 +184,19 @@ <h3 id="resource_wide_defaults">Resource-wide &amp; string-specific defaults</h3
 <p>Low-level support for <a>natural language</a> string metadata is widespread because the use of metadata for storage and interchange of the language of data values is long-established and widely supported in the basic infrastructure of the Web. This includes language attributes in [[XML]] and [[HTML]]; string types in schema languages (e.g. [[xmlschema11-2]]) or the various RDF specifications including [[JSON-LD]]; or protocol- or document format-specific provisions for language.</p>
 
 
+
+<div class="req" id="bp_default_setting">
+<p class="advisement">Specifications MAY define a rule or a field to provide the default language and base direction for all strings in a given resource. However, specifications MUST NOT assume that a resource-wide default is sufficient.  Even if a resource-wide setting is available, it must be possible to use string-specific metadata  to override that default.</p>
+</div>
+
+<p>Resource-wide defaults are particularly useful for increasing efficiency for any resources that use only a single language and have a consistent base text direction. It also reduces the overall complexity of a given document instance, since the language and direction values don't have to be repeated across many fields.</p>
+
+<p>However, there will always be exceptions to the default, and so it must be possible to override the default on a string-by-string basis, where necessary.</p>
+
+<p> First-strong heuristics are not applied to strings when the direction has been set externally using metadata. Even if an RLM character has been prepended to a string, the default metadata overrides it. Therefore it is necessary to use explicitly provided field data to override the default for strings whose base direction does not match the resource-wide default.</p>
```

Comment:
Remove extra space from front of paragraph.

I'm not sure the RLM sentence is helpful here: I know what you're trying to say, but the "therefore" doesn't really follow from that sentence. Perhaps rephrase as:

> First-strong heuristics are not applied to strings when the direction has been set externally using metadata. Attempting to override the resource-wide default direction by inserting strong directional markers (such as RLM) or using paired bidi controls is insufficient, since the paragraph base direction set in the metadata will still influence aspects of text processing (such as paragraph alignment). In addition, most specifications should avoid the need to modify string data. Therefore it is necessary to explicitly provide for metadata for individual string values to override the resource-wide default.

Reply:
Some changes made.